src/events/types.ts

```diff
@@ -50,7 +52,7 @@ export type ScheduledItemEventBusPayload = BaseEventBusPayload &
   Pick<ScheduledItemModel, 'createdBy' | 'scheduledSurfaceGuid'> &
   Pick<
     ApprovedItem,
-    'topic' | 'isSyndicated' | keyof Omit<CorpusItem, 'id' | 'authors'>
+    'topic' | 'isSyndicated' | keyof Omit<CorpusItem, 'id'>
```

Comment:
this is some deep typescripting 😆 . can you explain this reverted change to me?

Reply:
@jpetto, this went in in the initial PR that added schema changes and updated all the resolvers. I think what happened is that this was added to fix a TS error temporarily to let the rest of the code compile but then this part of the code was forgotten about and so this change went in unnoticed.